cmd/chainsimulator/main.go

```diff
@@ -137,9 +137,15 @@ func startChainSimulator(ctx *cli.Context) error {
 
 	startTimeUnix := ctx.GlobalInt64(startTime.Name)
 	apiConfigurator := api.NewFreePortAPIConfigurator("localhost")
+
+	tempDir := os.TempDir()
+	defer func() {
+		_ = os.RemoveAll(tempDir)
```

Comment:
is it ok to remove all in case it returns /tmp ?

Reply:
Good catch @sstanculeanu :+1: 
maybe use something from this example?
```GO
aaa, err := os.MkdirTemp(os.TempDir(), "chainSimulator")
assert.Nil(t, err)

fmt.Println(aaa)
```

Reply:
done